src/Provider/Nominatim/Nominatim.php

```diff
@@ -230,8 +230,8 @@ private function jsonResultToLocation(\stdClass $place, bool $reverse): Location
 
         $location = $builder->build(NominatimAddress::class);
         $location = $location->withAttribution($place->licence);
-        $location = $location->withOSMId(intval($place->osm_id));
-        $location = $location->withOSMType($place->osm_type);
+        $location = $location->withOSMId(intval($place->osm_id ?? NULL));
```

Comment:
```suggestion
        $location = $location->withOSMId(intval($place->osm_id ?? null));
```
Lowercase `null` please as per PSR2 (and below)

Reply:
I don't think the null fallback should be inside int cast.  Maybe something like:

```suggestion
if (!empty($place->osm_id) {
    $location = $location->withOSMId((int) $place->osm_id);
}
```

Reply:
`intval($place->osm_id ?? null)` is wrong anyway, it would set the `OSMId` to `0` which is not what we want!